tap_linkedin_ads/client.py

```diff
@@ -223,8 +265,9 @@ def check_accounts(self, config):
         factor=2
     )
     def request(self, method, url=None, path=None, **kwargs):
+        self.get_access_token()
         if not self.__verified:
-            self.__verified = self.check_access_token()
+            self.__verified = self.get_access_token()
```

Comment:
It's possible I might be missing something, but I don't think get_access_token returns a value. Also, would this do anything since we call `self.get_access_token()` on L268? If it wasn't verified then, would it be verified now?

Reply:
yes it does not return anything , i have corrected it